CONTRIBUTING.md

```diff
@@ -8,55 +10,77 @@ This project is built with [Lerna](https://lerna.js.org/) and contains the follo
 - `millicast-chromecast-receiver`: Example of Google Cast receiver for demo.
 
 ## Development
+
 Asumming that you have Node 12.10.x or newer and `npm` installed, install the required dependencies running:
+
 ```sh
 npm ci
 ```
 
 ### Changesets
+
 Whenever you are working on a new feature, fix or change, make sure you create a **changeset** with a description of the change. Follow [this](.changeset/README.md#Changes-per-feature) guide for further understanding.
 
 ### Building packages
+
 As the project is built using [Lerna](https://lerna.js.org/), we can rely on it to manage our packages dependencies, so you just need to run at project's root directory
 ```sh
 npm run prepare
 ```
 
-Next, to build all packages add a `.env` file in both demo packages (`millicast-publisher-demo`, `millicast-viewer-demo` & `millicast-chromecast-receiver`). You can find the following example in `.env.sample`:
+You will need to add a `.env` file in all demo packages (`packages/millicast-publisher-demo`, `packages/millicast-viewer-demo` & `packages/millicast-chromecast-receiver`). You can simply copy the `.env.sample` to get started:
```

Comment:
To include all we should add `packages/millicast-multiview-demo` and `packages/millicast-webaudio-delay-demo`

Reply:
Fixed thank you